ingenialink/canopen/network.py

```diff
@@ -266,8 +274,8 @@ def scan_slaves(self) -> List[int]:
         self._connection.scanner.reset()
         try:
             self._connection.scanner.search()
-        except Exception as e:
-            logger.error("Error searching for nodes. Exception: {}".format(e))
+        except Exception:
+            logger.exception("Error searching for nodes.")
```

Comment:
Why remove information from the exception?

Reply:
The exception is automatically logged.